randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -563,6 +586,10 @@ Extra - map_name: rm_a5c14
           Any of the following:
               Mid-Air Morph (Beginner) or Power Grip Wall
               Speed Booster and Shinesparking Tricks (Beginner)
+  > Pipe to The Tower
+      Any of the following:
+          Enabled Septogg Helpers or Hijump With Power Grip Wall
```

Comment:
Noticed the reason why this was at the pipe: Since i used this under the assumption that you're on the pipe, which is necessary for the septogg helper. Since that isn't the case now, you'd have to duplicate (almost) all a3 pipe requirements to here too